config/testgrids/conformance/conformance-all.yaml

```diff
@@ -322,8 +322,6 @@ test_groups:
   days_of_results: 7
   column_header:
   - configuration_value: k8s-build-version
-- name: ppc64le-unit
```

Comment:
not sure what is the right way here, to remove it or add the test group in the dashboard ppc64le above 
https://github.com/kubernetes/test-infra/blob/master/config/testgrids/conformance/conformance-all.yaml#L132

Reply:
True. Removal of this test group altogether shall miss the k8s unit testing on ppc64le.
Must find the appropriate place to add this test group and create a corresponding dashboard.
@mkumatag

Reply:
@adisky please revert this change; we only want to remove the SIG Node dashboards. The job belongs to conformance and we should leave their dashboards alone.